trillian/ctfe/ct_server/main.go

```diff
@@ -226,7 +227,10 @@ func main() {
 
 	// Bring up the HTTP server and serve until we get a signal not to.
 	srv := http.Server{Addr: *httpEndpoint, Handler: handler}
+	shutdownWG := new(sync.WaitGroup)
+	shutdownWG.Add(1)
```

Comment:
Please move this to the inside of the function, right before defer `Done()`. Currently, it is equivalent to the previous code with channels (won't terminate unless a signal fires).